pkg/image/qemu_test.go

```diff
@@ -181,6 +183,15 @@ var _ = Describe("Report Progress", func() {
 	})
 })
 
+var _ = Describe("quantity to qemu", func() {
+	It("Should properly parse quantity to qemu", func() {
+		result := quantityToQemu(resource.MustParse("1Gi"))
```

Comment:
Never hurts to test go's strings.Replace function I suppose.

Reply:
Well we are testing that I am correctly stripping i and lowercasing k.